### ISSUE TITLE
Create script to extract clip from MP4 and save to S3

### ISSUE THREAD

**Issue by monocongo** [monocongo/video_utils#1]
Create a script extract a clip from an MP4 file and save this clip to an S3 bucket.

- [x] command line arguments for original video (MP4) file, start time, end time, and S3 bucket location
- [x] extraction of sub-clip from original MP4 file into new MP4 file
- [x] push of sub-clip MP4 to S3 bucket
- [x] document as much as possible in README

**Comment by monocongo** [comment]
Run the script like so:

```
$ python save_extract.py --source /home/james/video/big_buck_bunny_720p_2mb.mp4 --dest_s3_bucket elasticbeanstalk-us-east-2-867324276890 --dest_s3_key bunny_clip.mp4 --start 2 --end 7
```

**Comment by monocongo** [comment]
Resolved by PR #2
